guava/src/com/google/common/escape/ArrayBasedCharEscaper.java

```diff
@@ -18,6 +18,9 @@
 
 import com.google.common.annotations.Beta;
 import com.google.common.annotations.GwtCompatible;
+import org.checkerframework.checker.index.qual.LTEqLengthOf;
+import org.checkerframework.checker.index.qual.LengthOf;
+
```

Comment:
In guava, imports should be formatted according to the [google style guide](https://google.github.io/styleguide/javaguide.html#s3.3-import-statements) (don't use `*`, sort alphabetically, don't insert blank lines).

Reply:
That being said, this has low priority and can be easily fixed when everything else is ready.